src/client/static/hydration.tsx

```diff
@@ -11,10 +15,24 @@ export const hydrateApp = () => {
 
   const clientState = routingConfig.clientState;
 
+  const {
+    abTesting: { mvtId = 0, forcedTestVariants = {} } = {},
+  } = clientState;
+
   hydrate(
-    <StaticRouter location={`${routingConfig.location}`} context={{}}>
-      <Main {...clientState} />
-    </StaticRouter>,
+    <ABProvider
+      arrayOfTestObjects={tests}
+      abTestSwitches={switches}
+      pageIsSensitive={false}
+      mvtMaxValue={1000000}
```

Comment:
I assume this has to be the same value as that of the mvt config in vcl ? is it worth noting this as a comment so people don't accidentally change it.